modules/notifications/notifications.php

```diff
@@ -252,29 +244,19 @@ public function send_emails( $recipients, $subject, $message, $message_headers =
 	/**
 	 * Schedule a webhook notification
 	 *
-	 * @param string $current_user Current user's name
-	 * @param string $post_type Post type
-	 * @param int $post_id Post ID
-	 * @param string $edit_link Edit link for the post
-	 * @param string $post_title Post title
-	 * @param string $old_status Old status of the post
-	 * @param string $new_status New status of the post
-	 * @param string $post_timestamp Timestamp of the post's last update
+	 * @param string $webhook_message Message to be sent to webhook
+	 * @param string $action Action being taken, eg. status-change
+	 * @param string $timestamp Timestamp of the message, eg. the time at which the post was updated
 	 */
-	public function schedule_webhook_notification( $current_user, $post_type, $post_id, $edit_link, $post_title, $old_status, $new_status, $post_timestamp ) {
+	public static function schedule_webhook_notification( string $webhook_message, string $action, string $timestamp ): void {
```

Comment:
Good thinking precomputing these values to make this monster function signature smaller.

Reply:
Yip, makes it easier to use for the approve/reject code too